workflows/files/ZWEWRF06.xml

```diff
@@ -99,9 +99,8 @@
         <canMarkAsFailed>false</canMarkAsFailed>
         <template>
             <inlineTemplate substitution="true">//ALLOCDSS EXEC PGM=IEFBR14
-//LOADLIB1 DD DISP=(MOD,CATLG,DELETE),
-//    DSN=${instance-zowe_dsn_prefix}.LOADLIB,
-//    DSNTYPE=LIBRARY,SPACE=(CYL,(10,5,5)),
+//LOADLIB1 DD DISP=SHR,
+//    DSN=${instance-zowe_dsn_prefix}.SZWEAUTH
 #if(${instance-xmem_volume} and $!{instance-xmem_volume} != "")
```

Comment:
This will cause JCL error when specifying a volume, as `DSN=...` does not end with a comma. If you want to have a conditional `VOL=SER=...` definition it must be between the `DISPF=SHR,` and `DSN=...`. That said, specifying `VOL=SER=...` with `DISP=SHR` is only done for non-cataloged datasets. Further down you use `LISTDS` which requires that the dataset is cataloged, so I would just trash the conditional volser ref for SZWEAUTH here.

Reply:
Hi Onno. Yeah, I did that but forgot to hit save before the commit :D.